### ISSUE TITLE
[stable/homeassistant] Homeassistant fails to start with AppDaemon enabled

### ISSUE THREAD

**Issue by rwlove** [helm/charts#22742]
**Describe the bug**
Homeassistant fails to start with AppDaemon enabled.

**Version of Helm and Kubernetes**:
Helm: v3.2.1
Kubernetes: v1.18.2

**Which chart**:
stable/homeassistant

**What happened**:
```
➜  ~ kubectl -n homeassistant logs homeassistant-home-assistant-7d5d949c68-zqjfd -c appdaemon 
2020-06-10 18:45:03.650536 INFO AppDaemon Version 3.0.5 starting
2020-06-10 18:45:03.650939 INFO Configuration read from: /conf/appdaemon.yaml
2020-06-10 18:45:03.653758 INFO AppDaemon: Starting Apps
2020-06-10 18:45:03.657716 INFO AppDaemon: Loading Plugin HASS using class HassPlugin from module hassplugin
2020-06-10 18:45:03.724632 INFO AppDaemon: HASS: HASS Plugin Initializing
2020-06-10 18:45:03.725158 INFO AppDaemon: HASS: HASS Plugin initialization complete
2020-06-10 18:45:03.725450 INFO Starting Dashboards
2020-06-10 18:45:03.725692 WARNING ------------------------------------------------------------
2020-06-10 18:45:03.725788 WARNING Unexpected error during run()
2020-06-10 18:45:03.725884 WARNING ------------------------------------------------------------
2020-06-10 18:45:03.726918 WARNING Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/appdaemon/admain.py", line 82, in run
    self.rundash = rundash.RunDash(self.AD, loop, self.logger, self.access, **hadashboard)
  File "/usr/local/lib/python3.6/site-packages/appdaemon/rundash.py", line 130, in __init__
    dash_net = url.netloc.split(":")
TypeError: a bytes-like object is required, not 'str'

2020-06-10 18:45:03.727020 WARNING ------------------------------------------------------------
2020-06-10 18:45:03.727147 INFO AppDeamon Exited
```
**What you expected to happen**:
AppDaemon starts.
Homeassistant starts.

**How to reproduce it** (as minimally and precisely as possible):
```
appdaemon:
  enabled: true

  ## code-server container image                                                                                                                                              
  ##                                                                                                                                                                          
  image:
    repository: acockburn/appdaemon
    tag: 3.0.5
    pullPolicy: IfNotPresent

  ## Home Assistant API token                                                                                                                                                 
  # haToken:                                                                                                                                                                  

  ## Additional hass-vscode container environment variable                                                                                                                    
  ## For instance to add a http_proxy                                                                                                                                         
  ##                                                                                                                                                                          
  extraEnv: {}

  ingress:
    enabled: false
    annotations: {}
      # kubernetes.io/ingress.class: nginx                                                                                                                                    
      # kubernetes.io/tls-acme: "true"                                                                                                                                        
    path: /
    hosts:
      - appdaemon.local
    tls: []
    #  - secretName: appdaemon-tls                                                                                                                                            
    #    hosts:                                                                                                                                                               
    #      - appdaemon.local                                                                                                                                                  

  service:
    type: ClusterIP
    port: 5050
    annotations: {}                                                                                                                                                           
    labels: {}                                                                                                                                                                
    clusterIP: ""
    ## List of IP addresses at which the hass-appdaemon service is available                                                                                                  
    ## Ref: https://kubernetes.io/docs/user-guide/services/#external-ips                                                                                                      
    ##                                                                                                                                                                        
    externalIPs: []
    loadBalancerIP: ""
    loadBalancerSourceRanges: []
    # nodePort: 30000
```

**Comment by kifeo** [comment]
I have the exact same issue, thanks for reporting

**Comment by rwlove** [comment]
@kifeo After further investigation is seems that stable/home-assistant is deprecated (see the README.md).

I tried running with https://github.com/billimek/billimek-charts/tree/master/charts/home-assistant, which is what stable/home-assistant's README.md directs users to. Unfortunately, I hit the same bug.

I've filed a defect against billimek, here. You might want to try with this chart and comment on my bug report if you still have the same problem, as I do.
https://github.com/billimek/billimek-charts/issues/248
